external-api/src/cron/eStoreSiteCreationJob.cron.ts

```diff
@@ -79,17 +100,21 @@ export const eStoreSiteCreationCron = async (eStoreData: Estore): Promise<void>
     );
   } else {
     // Step 3: Site creation has failed
+
+    // Stop CRON job
+    cron(eStoreData, ENDPOINT.SITE, true);
```

Comment:
Probably not for this PR, but I wasn't sure what `true` meant here - this could be updated to have object structured params so it's easier to digest (IMO) e.g

```ts
cron({
  eStoreData,
  category: ENDPOINT.SITE,
  kill: true,
});
```